### DISCUSSION TITLE
[Help me] Quick Launch & Tray Icon on secondary task bar have disappeared today

### DISCUSSION THREAD

**Discussion by TaengHee** [valinet/ExplorerPatcher#1630] [Q&A]
First, I am sorry I am not good at English.

Quick Launch & Tray Icon on secondary task bar have disappeared today.
They were on both primary & secondary task bar untill yesterday.
However, when I turned of my PC today, I found they're gone on secondary task bar.

So I checked if updated version was installed automatically, but it was not.

I don't know why they are gone suddenly.
I am so sad
Please, help me.

Thank you

**Comment by pyrates999** [comment]
Try rebooting.

Then please post windows version and build.
Please post explorer patcher version.

**Reply by TaengHee** [reply]
Really glad & happy to your comment.
I am sorry for checking your comment late.

I tried reboot but it is same.

Window version is Windows 11 Home (22H2)
Build is 22621.1992
Explorer Patcher version is 22621.1555.55.1

Could you help me??

**Comment by pyrates999** [comment]
Try the following:

1. uninstall explorer patcher
2. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. reboot
4. install explorer patcher
5. verify you get a notification that symbols have been downloaded
6. restart windows explorer using either the task manager or in the properties window of explorer patcher
7. reboot

Then apply your settings again in explorer patcher to see if it fixes it.

**Reply by TaengHee** [reply]
Thank you so much for your commnet.

I tried step 1 to 4 but I can't understand 5 and 6 exactly.

Is 'a notifiation' of step 5 this message ↓ that appears on right-bottom after Explorer Patcher is installed??
─────────────────────────────────
Version 22621.1555.55.1 is available 
You can udate by right clicking the taskbar,
choosing "Properties", then "Updates'. Click here to
learn more about this udpate.
─────────────────────────────────
If I saw this message, is 5 done??

And step 6.
How can I restart windows explorer using either the task manager or in the properties window of explorer patchter??
What should I click??

Could you help me again??

**Comment by pyrates999** [comment]
You installed an older version of explorer patcher.  Uninstall it and download the current release from here.

**Comment by highlanderc** [comment]
This didn't fix it for me.
